data/config-schema/src/main/resources/net/corda/schema/configuration/messaging/1.0/corda.messaging.json

```diff
@@ -99,6 +99,11 @@
       "description": "Settings for publishers that write messages to the underlying bus",
       "$ref": "publisher.json",
       "default": {}
+    },
+    "maxAllowedMessageSize": {
+      "description": "Maxiumum allowed size, in bytes, for publishing Messages. This configuration does _not_ change the Kafka configuration itself.",
+      "type": "number",
+      "default": 972800
```

Comment:
Wonder if it's worth adding some sort of minimum (although I'm not totally sure what that should be) - I expect we'd get some instability if this is set too small.

Reply:
Good point. I added both minimum and maximum.
We can review both values, but I choose 500kb as a minimum as I thought going small may compromise the performance of the platform, and I picked 8Mb as the maximum as that's what we currently have as the size of the chunks column in the DB.